src/components/Program/Program.css

```diff
@@ -0,0 +1,86 @@
+.program {
+display: flex;
+background-image: url('/public/images/program.png');
+background-size: cover;
+background-position: center;
+flex-direction: column;
+}
+
+.program h1 {
+display: flex;
+flex-wrap: wrap;
+justify-content: center;
+align-items: center;
+font-family: "Poppins", sans-serif;
+font-weight: 500;
+font-size: 48px;
+margin-bottom: 20px;
+}
+
+.program-images {
+display: flex;
+flex-wrap: wrap;
+justify-content: center;
+}
+
+.program-image {
+display: flex;
+flex-direction: column;
+align-items: center;
+width: 45%;
+margin-bottom: 30px;
+}
```

Comment:
These images are not responsive in tablet and mobile. I think a specific width is needed to fix this.